### ISSUE TITLE
Two GPU train error occurred:block: [73,0,0], thread: [30,0,0] Assertion input_val >= zero && input_val <= one failed. 

### ISSUE THREAD

**Issue by ZhenxinYUAN** [Megvii-BaseDetection/YOLOX#748]
train:  python tools/train.py -f exps/default/yolox_s.py -d 2 -b 32 -c yolox_s.weights
GPU : 2* 2080TI
datasets: COCO2017

Error:
/pytorch/aten/src/ATen/native/cuda/Loss.cu:102: operator(): block: [73,0,0], thread: [30,0,0] Assertion input_val >= zero && input_val <= one failed.

THCudaCheck FAIL file=/pytorch/aten/src/THC/THCCachingHostAllocator.cpp line=278 error=710 : device-side assert triggered

I tried to set the Batch_size low, but failed

**Comment by leetesua** [comment]
did you solved the problem?
I got the same problem, how did you solve it

**Comment by ZhenxinYUAN** [comment]
> did you solved the problem? I got the same problem, how did you solve it

You can try to lower the value of batch_size, such as  set -b 16

**Comment by ToniButland1998** [comment]
> > did you solved the problem? I got the same problem, how did you solve it
> 
> You can try to lower the value of batch_size, such as set -b 16

Do we have any better ways to solve it, besides just lower batch size? itried make lr 10 times smaller; and it only works when batch size is 8;

But 8 is not a reasonal batch size number for me.

So any other approaches can solve it? thanks.
